api-docs/leads.md

```diff
@@ -7,3 +7,85 @@ Here is the list of the valid log types:
 - `leadCreated`,
```

Comment:
I am thinking we should actually get rid of this now -- too much work to create a whole leadLogs database table and then copy things over into the clientLogs table.

We can have a clientLog event for when the lead is converted to a client, but i don't think we should do the intensive audit log stuff for leads that we do for clients.